ices/81270.rs

```diff
@@ -0,0 +1,27 @@
+#![feature(allocator_api)]
```

Comment:
Could you use minimized example (https://github.com/rust-lang/rust/issues/81270#issuecomment-765403789) instead?

Reply:
I have assassinated the culprit: it was `rustfmt` and the "format on save" setting.